tcms/testplans/templates/testplans/get.html

```diff
@@ -140,6 +140,32 @@ <h2 class="card-pf-title">
     </div>
 </template>
 
+<template id="test_case_row_details">
+    <div class="list-view-pf-left">
+        <a href="/case/${testCase.id}" class="js-test-case-link"></a>
```

Comment:
Broken href. Use empty value and adjust the attribute value in getTestCaseRowContent. Use `/` at the end of the URL.